pkg/blackduck/ctl_blackduck.go

```diff
@@ -65,7 +65,8 @@ type Ctl struct {
 	ProxyCertificateFilePath              string
 	AuthCustomCAFilePath                  string
 	Type                                  string
-	DesiredState                          string
+	Start                                 bool
```

Comment:
Leave DesiredState in the CRD/ctl structs.  Only thing that needs to change is the outward appearance.  --stop and --start just set the desiredState field appropriately.